content/projects/basic-unit-test-mocks-kata/_index.md

```diff
@@ -153,4 +153,16 @@ then they are wrong.
 - You can make the emails that get sent look good as well. it doesn't just need to be plain text
 - You can look into sending sms instead of email
 - link to the quote author's wikipedia page straight from the email/sms
-- add extra information such as a weather report
\ No newline at end of file
+- add extra information such as a weather report
+
+**Instructions for reviewer**
+
+- Please pay attention to how the unit tests are written:
+
+> They should demonstrate that the code does what it should and does it efficiently.
+
+> They should test correctly. Please refer to : https://umuzi-org.github.io/tech-department/topics/jasmine-spies/
```

Comment:
please dont hardcode urls. What if we rename the topic or move it? Then the link breaks

Use a contentlink rather. Then at least the linter will tell us about the broken link so we can fix it.